low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -107,7 +107,7 @@ def copy_input_config(self, stage=None):
         if not os.path.exists(path) or stage == "post_install":
             with open(path, "w") as f:
                 f.write("")
-        Conf.load(GLOBAL_CONFIG_INDEX, global_config_path)
+        Conf.load(GLOBAL_CONFIG_INDEX, global_config_path, fail_reload=False)
```

Comment:
> fail_reload
what this means really? We need to reload basically

Reply:
fail_reload: If key is already loaded, reloading will not throw an error.
Ref.: [ConfStore](https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/316342315/Conf+Store#Appendix-B%3A-ConfStore-Interfaces)